src/test/java/kitchenpos/bo/OrderBoTest.java

```diff
@@ -0,0 +1,125 @@
+package kitchenpos.bo;
+
+import kitchenpos.Fixtures;
+import kitchenpos.dao.MenuDao;
+import kitchenpos.dao.OrderDao;
+import kitchenpos.dao.OrderLineItemDao;
+import kitchenpos.dao.OrderTableDao;
+import kitchenpos.model.Order;
+import kitchenpos.model.OrderLineItem;
+import kitchenpos.model.OrderStatus;
+import kitchenpos.model.OrderTable;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+class OrderBoTest {
+
+    @Mock
+    private MenuDao menuDao;
+    @Mock
+    private OrderDao orderDao;
+    @Mock
+    private OrderLineItemDao orderLineItemDao;
+    @Mock
+    private OrderTableDao orderTableDao;
+
+    @InjectMocks
+    OrderBo orderBo;
+
+    private Order defaultOrder;
+    private List<OrderLineItem> defaultOrderLineItems;
+    private List<Long> defaultMenuIdsOfOrderLineItems;
+    private OrderTable defaultOrderTable;
+
+    @BeforeEach
+    public void setUP() {
+        defaultOrder = Fixtures.getOrder(1L, LocalDateTime.now(), getDefaultOrderLineItem(), OrderStatus.MEAL.name(), 1L);
+        defaultOrderLineItems = getDefaultOrderLineItem();
+        defaultMenuIdsOfOrderLineItems = defaultOrderLineItems.stream()
+                .map(OrderLineItem::getMenuId)
+                .collect(Collectors.toList());
+        defaultOrderTable = Fixtures.getOrderTable(1L, false, 4);
+    }
+
+    @DisplayName("정상적인 값으로 주문이 생성된다.")
+    @Test
+    public void createNormal() {
+        given(menuDao.countByIdIn(defaultMenuIdsOfOrderLineItems)).willReturn(Long.valueOf(defaultOrderLineItems.size()));
+        given(orderTableDao.findById(defaultOrder.getOrderTableId())).willReturn(Optional.ofNullable(defaultOrderTable));
+        given(orderDao.save(defaultOrder)).willReturn(defaultOrder);
+
+        assertThat(orderBo.create(defaultOrder)).isEqualTo(defaultOrder);
```

Comment:
given, when, then을 완벽하게 나눠보는건 어떨까요??^^

```suggestion
        Order order = orderBo.create(defaultOrder);

        assertThat(order).isEqualTo(defaultOrder);
```

given-when-then이 명확해질것 같습니다.^^